app/locale/zh-HANS.js

```diff
@@ -857,6 +857,7 @@ module.exports = {
       cancel: '取消',
       save: '保存',
       publish: '发布',
+      output: '输出',
```

Comment:
why do u need to add string? Generally it's uncommenting stuff. Is that causing testcase to fail?